assets/js/admin.js

```diff
@@ -25,7 +25,6 @@ const Admin = {
     Admin.setup_xeditable(subject);
     Admin.setup_inline_form_errors(subject);
     Admin.setup_tree_view(subject);
-    Admin.setup_sticky_elements(subject);
```

Comment:
This breaks again one of my applications :(

IMHO: We must keep the method names (with empty content)

Reply:
> This breaks again one of my applications :(

How ?

> IMHO: We must keep the method names (with empty content)

Seems like a good idea indeed.

@onEXHovia or @core23 ; is someone ok to make the pr ?

Reply:
> > > IMHO: We must keep the method names (with empty content)
> > 
> > 
> > Seems like a good idea indeed.
> > @onEXHovia or @core23 ; is someone ok to make the pr ?
> 
> I can, but first I would like to understand the context

I was using some code like to recreate the sticky functionality for some react component inside the admin. 

```
    if (window.Admin !== undefined) {
      // @ts-ignore
      window.Admin.setup_sticky_elements(document);
    }
```

As the `setup_sticky_elements` method was removed my whole react script breaks and does not load anymore.

Reply:
I wonder if the whole JS code shouldn't have been considered as internal... 
But it's maybe too late now.

Reply:
I believe we don't need to strictly maintain BC as we do for the backend. Projects will need to update their codebase anyway and move away from using jquery.

We're releasing minor updates in terms of scope, but in some cases, projects will still need to update their js.

Suppose we've replaced the readmore-js jquery plugin to stimulus controller. However, some projects might have been using it directly. In that case, they'll need to add readmore-js to their package.json continue using it, since sonata only provides collapse functionality, and readmore-js is an internal library used to implement that.